### PR TITLE
Add new Automation API version

### PR DESCRIPTION
**Pull request by katherinebecker**
This PR adds a new GA Automation API version for Update Management resources. The new API version adds a paged response when listing software update configurations. It also includes fixes to the description text and the response schema for listing software update configurations.

### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [x] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [x] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Review by anthony-c-martin — APPROVED** [review]
Thanks for taking the time & care to organize the commits so clearly, and fixing up all the grammar/spelling issues! :)

**Comment by xccc-msft** [comment]
@vrdmr please review this PR as the code owner

File: specification/automation/resource-manager/Microsoft.Automation/stable/2019-06-01/softwareUpdateConfigurationRun.json
Comment (vrdmr) [inline specification/automation/resource-manager/Microsoft.Automation/stable/2019-06-01/softwareUpdateConfigurationRun.json]: You might want to verify these URLs. This one does not work.

**Review by vrdmr — APPROVED** [review]
Minor comment. Everything else LGTM.

Just to make sure that https://github.com/Azure/azure-resource-manager-schemas gets updated. (It's been couple of years I've worked on this now and don't know the current proces)

**Comment by xccc-msft** [comment]
@katherinebecker can you verify this url as per minor comment?

**Comment by xccc-msft** [comment]
get confirmed from @katherinebecker , hold this PR until it's fixed.

**Comment by xccc-msft** [comment]
@katherinebecker any progress?

**Comment by katherinebecker** [comment]
@xccc-msft, I've fixed the URLs. We still need to wait on merging this PR because the new API version is not available yet. We are waiting on a deployment.

**Comment by xccc-msft** [comment]
@katherinebecker can we merge this first and open new PR when your new API version available?

**Comment by katherinebecker** [comment]
@xccc-msft, my understanding was that our documentation would be updated automatically after merging. Since we don't want the documentation showing an API version that isn't available yet, we'd need to wait before merging the PR. Is this correct?

**Comment by xccc-msft** [comment]
@katherinebecker I'm not expert on this. Let's hold this PR until your side is ready. For now, I just add label of 'DoNotMerge' for the onhold status.

**Comment by akning-ms** [comment]
Hi @katherinebecker， Is your service ready? as this PR is existing for a long time, can you close it?

**Review by assing — APPROVED** [review]

**Comment by katherinebecker** [comment]
@akning-ms, this PR can be merged now

File: specification/automation/resource-manager/Microsoft.Automation/stable/2019-06-01/examples/softwareUpdateConfigurationMachineRun/listSoftwareUpdateConfigurationMachineRunsByRun.json
Comment (assing) [inline specification/automation/resource-manager/Microsoft.Automation/stable/2019-06-01/examples/softwareUpdateConfigurationMachineRun/listSoftwareUpdateConfigurationMachineRunsByRun.json:62]: Resource name should be softwareUpdateConfigurationMachineRuns

File: specification/automation/resource-manager/Microsoft.Automation/stable/2019-06-01/examples/softwareUpdateConfigurationMachineRun/listSoftwareUpdateConfigurationMachineRuns.json
Comment (assing) [inline specification/automation/resource-manager/Microsoft.Automation/stable/2019-06-01/examples/softwareUpdateConfigurationMachineRun/listSoftwareUpdateConfigurationMachineRuns.json:60]: Resource name should be softwareUpdateConfigurationMachineRuns

**Comment by xccc-msft** [comment]
Close this now as no update action taken recently. Please feel free to reopen if you still need to work on this PR. Thanks.

**Pull request closed without merging**
